atlas-search/introduction.mdx

```diff
@@ -1,93 +1,102 @@
+* [Content Data Sync](#content-data-sync) - How to clone/ sync WordPress data into Atlas Search
+* [Using Atlas Search](#using-atlas-search) - A series of sample GraphQL queries that demonstrate Atlas Search’s functionalities.
     + [Querying Standard WP Data Types](#querying-standard-wp-data-types-) WP Data Types - Queries with Posts, Pages, CPT
     + [Querying ACM/ ACF Types](#querying-acm-or-acf-types-) - Queries for Atlas Content Modeller or Advanced Custom Fields
     + [Getting More Specific with Queries](#getting-more-specific-with-queries) - Queries with `AND`, `NOT` & `OR`
 * [Advanced Configuration of Atlas Search](#advanced-configuration-of-atlas-search) - Configure what WP Data Types & fields are searchable, & set custom weightings to prioritize what matters to you.
-* [Feedback](#feedback) - Let us know what you think of Atlas Search or if you are having any issues.
   
   
 
-### Prerequisites
-#### General Prerequisites:
-1. You are an existing or “eval” Atlas customer.
-2. You have an instance of WordPress on a WPEngine ‘Portal’ ‘site’ with a target ‘environment’ & you can access that environment’s ‘WP Admin’ panel.
-3. In the target environment’s WP Admin panel, you have downloaded & activated the [WP-GraphQL](https://wordpress.org/plugins/wp-graphql/) plugin from the WordPress.org plugin repository.
+### Getting an Atlas Search Account
+To get started with Atlas Search, you will need to fill in the [enrolment form](https://docs.google.com/forms/d/e/1FAIpQLSeRdfLXHlAhjaBfMJOobynCwxuKFtWwzHzna_yRXFt0n_dbhA/viewform).
+Please note that this is limited to 1 enrollment per email.
+After submission of the form you will receive an Atlas Search 'setup' email with the your own personal provider backend URL & Access Token. **Do not share this setup email or the Atlas Search credentials**
 
-#### Atlas Search Prerequisites (Unless configured via ‘Atlas Blueprints’):
-1. You have enrolled for Atlas Search, via:
-    + The [internal enrolment form (WPEngine employees only)](https://docs.google.com/forms/d/e/1FAIpQLSeq4pK2PLUouEk1v_Z7EgSPqPU-H5PNdI2j6FYWhLtF2jQyQA/viewform)
-    + The [external enrolment form (External Customers)](https://docs.google.com/forms/d/e/1FAIpQLSeRdfLXHlAhjaBfMJOobynCwxuKFtWwzHzna_yRXFt0n_dbhA/viewform):
-        - Requires Gmail account.
-        - Limited to 1 instance per Gmail account.
-    + Automatic-enrollment as part of Atlas “eval” account.
-2. You have received an Atlas Search setup email with the following:
-    + A link to this documentation
-    + A link to download the latest [Atlas Search](https://wordpress.org/plugins/atlas-search/) plugin, from WP Plugin Store.
-    + Your own personal Atlas-site URL and Access Token.
+### Installing the Atlas Search Plugin
+The Atlas Search plugin can be installed manually but will more likely be installed implicitly as part of our Atlas Blueprints setup.
   
+#### Installation via Blueprints
```

Comment:
I think it's good to break out the blueprint vs. non-blueprint users here for instructions